docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -259,11 +261,25 @@ Custom Metrics
 ==============
 You can also pass your own custom metric to relevant checks and suites.
 
+For tabular metrics the custom metrics should support the
+`sklearn scorer <https://scikit-learn.org/stable/modules/generated/sklearn.metrics.make_scorer.html>`__ API, which is
+a function that accepts the parameters: (model, x, y_true), and returns a score with the convention that higher is
+better.
+
 For computer vision the custom metrics should support the
-`Ignite Metric <https://pytorch.org/ignite/metrics.html#how-to-create-a-custom-metric>`__ API.
+`Ignite Metric <https://pytorch.org/ignite/metrics.html#how-to-create-a-custom-metric>`__ API. Ignite Metrics are
+classes that inherit from the ``Metric`` class and implement the ``reset``, ``update`` and ``compute`` methods.
+The ``update`` method is called on each batch of data, and the ``compute`` method is called to compute the final metric.
+Note that contrary to the natively implemented ignite metrics, which expect the the labels and predictions to be
+pytorch tensors, your custom metrics should expect lists of numpy objets. For example, for classification the labels
+will be a list of integers, while the predictions will be lists of numpy array representing the softmax outputs. This is
+designed to enable deepcheck to function regardless of the underlying framework.
+
+Fir the same reason, for classification you shouldn't implement custom Ignite Metrics, but rather use the (simpler)
+Scikit-learn Scorer API which deepchecks supports natively also for computer vision tasks. Ignite metrics are
+recommended only if you haven't found a suitable metric in the list of deepchecks metrics above, or for new tasks that
+are not yet supported by deepchecks.
```

Comment:
maybe describe CustomClassificationScorer which is a useful for custom classification scorers

Reply:
alternative is to describe here you can just send a custom callable + edit stuff in code - but i guess we do not want to do that